### PR TITLE
Add flags for event storage limits

### PR DESCRIPTION
**Pull request by vmarmol**
/cc @rjnagal

### CODE REVIEW

File: manager/manager.go [hunk 1]
```diff
@@ -949,3 +951,50 @@ func (self *manager) GetPastEvents(request *events.Request) ([]*info.Event, erro
...
+		}
+		policy.PerTypeMaxAge[info.EventType(items[0])] = dur
+	}
+
+	// Parse max number.
+	parts = strings.Split(*eventStorageEventLimit, ",")
+	for _, part := range parts {
+		items := strings.Split(part, "=")
+		if len(items) != 2 {
+			glog.Warningf("Unknown event storage policy %q when parsing max event limit")
```
Comment (rjnagal) [inline manager/manager.go]: missing arg.

File: manager/manager.go [hunk 2]
```diff
@@ -949,3 +951,50 @@ func (self *manager) GetPastEvents(request *events.Request) ([]*info.Event, erro
...
+// Parses the events StoragePolicy from the flags.
+func parseEventsStoragePolicy() events.StoragePolicy {
+	policy := events.DefaultStoragePolicy()
+
+	// Parse max age.
+	parts := strings.Split(*eventStorageAgeLimit, ",")
+	for _, part := range parts {
+		items := strings.Split(part, "=")
+		if len(items) != 2 {
+			glog.Warningf("Unknown event storage policy %q when parsing max age")
```
Comment (rjnagal) [inline manager/manager.go]: missing arg

**Comment by rjnagal** [comment]
LGTM, with minor nits.

File: manager/manager.go (hunk 1 above)
Comment (vmarmol) [inline manager/manager.go]: Done.

File: manager/manager.go (hunk 2 above)
Comment (vmarmol) [inline manager/manager.go]: Done.

**Comment by vmarmol** [comment]
Thanks @rjnagal! PTAL

**Pull request merged**